bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -75,6 +75,10 @@ type KubeadmConfigSpec struct {
 	// +optional
 	Mounts []MountPoints `json:"mounts,omitempty"`
 
+	// BootCommands specifies extra commands to run very early in the boot process
+	// +optional
+	BootCommands []BootCommand `json:"bootCommands,omitempty"`
+
 	// PreKubeadmCommands specifies extra commands to run before kubeadm runs
 	// +optional
```

Comment:
```suggestion
	// PreKubeadmCommands specifies extra commands to run before kubeadm runs. 
	// With cloud-init, this is prepended to the runcmd module configuration, and is typically executed in
	// the cloud-final.service systemd unit. In Ignition, this is prepended to /etc/kubeadm.sh.
	// +optional
```

And repeat for PostKubeadmCommands, replacing "prepended" with "appended".